Adyen.xcodeproj/project.pbxproj

```diff
@@ -5965,9 +5954,9 @@
 				);
 				MARKETING_VERSION = 4.6.1;
 				OTHER_SWIFT_FLAGS = "-Xfrontend -warn-long-expression-type-checking=100";
-				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.AdyenUIHost;
+				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.CheckoutDemoUIKit;
```

Comment:
Don't commit this

Reply:
well, we will have to switch it every time locally to debug, right 🤔

Reply:
It shouldn't any effect at all. How does it alter debugging?

Reply:
ok, I guess you right.
Distribution certificate doesn't mean that we can't build it in DEBUG mode..